app/theme/client/main.css

```diff
@@ -58,3 +58,429 @@
 
 /* RTL */
 @import 'imports/general/rtl.css';
+
+/* customerChat History CSS */
+.inner-addon {
+	position: relative;
+            left: 24px;
+
+                    width: 345px;
+                        height: 40px;
+
+                        box-sizing: border-box;
+                
+                    border: 2px solid #cbced1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5378317)